### PR TITLE
Change loadMap.py to check corrupted contents from file

### PR DESCRIPTION
**Pull request by youn0125**
Add check logic in loadMap.py
Add map files (noDiamond, containsOutsideCells, both)

Signed-off-by: Mi Yon Kim <youn0125@gmail.com>

### CODE REVIEW

File: maps/raw/outsideCell.map [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Axe
+Shears
+Pretty Rock
+######################
+12,12,1,1,None
+13,12,0,1,Tree
+14,12,0,2,None
+12,13,0,1,None
+12,14,0,0,Royal Diamonds
+27,14,0,0,None
```
Comment (DanielBrewerPsu) [inline maps/raw/outsideCell.map]: It would be good to have tests for the extreme "edge conditions". that's when a coordinate is <1 or >25. 27 is a good number, but 26 would be better. I'll add a raw map file that tests this in a separate feature branch.
